docs/reference/errors-and-warnings/NU1604.md

```diff
@@ -12,6 +12,35 @@ f1_keywords:
 
 # NuGet Warning NU1604
 
+## Missing Package Version
+
+> Project dependency 'PackageA' does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results.
+
+### Issue
+
+A project dependency doesn't define a version.<br/><br/>This means that restore used the lowest available version. Each restore will float downwards trying to find a lower version that can be used. This means that restore goes online to check all sources each time instead of using the packages that already exist in the user package folder.
```

Comment:
You don't need to add `<br/>` html, just put an empty line and let the markdown renderer convert it.

Also, please put one sentence per line. This makes it easier to comment on specific sentences, but github only highlights diffs when a line is shorter than a certain number of characters. Multi-sentence lines almost always exceed the limit, making future pull requests hard to review.

Reply:
For the record, that text was already there in that format, I'll update it though, its just going to make the diff bigger